src/conn/conn_cache.c

```diff
@@ -380,32 +115,9 @@ __wti_cache_stats_update(WT_SESSION_IMPL *session)
     WT_STATP_CONN_SET(session, stats, cache_bytes_other, __wt_cache_bytes_other(cache));
```

Comment:
Sanity check: A few lines above `__wt_cache_pages_inuse()` uses `cache->pages_inmem` and `cache->pages_evicted`.
Does `pages_evicted` belong to evict or to cache? If it belongs to evict the new `WT_EVICT` struct won't be allocated yet during the `wiredtiger_open` flow.

Reply:
`pages_evicted` belongs in `cache`, as it is part of the `cache metadata`.